NAMESPACE

```diff
@@ -1,13 +1,14 @@
+# Generated by roxygen2: do not edit by hand
+
 S3method(gtrends,default)
 S3method(plot,gtrends)
 S3method(summary,gtrends)
 export(gconnect)
 export(gtrends)
-import(RColorBrewer)
 import(RCurl)
 import(ggplot2)
 import(googleVis)
 import(zoo)
-importFrom("graphics", "plot")
-importFrom("stats", "na.omit", "reshape")
-importFrom("utils", "data", "read.csv")
```

Comment:
I have no strong feeling and kinda prefer it without the quotes.  But when I ran R-devel with `R CMD check` it suggested them copy-and-paste ready with quotes.  Just so you know :)